client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -201,15 +188,6 @@ export const setSiteTitle = ( siteTitle: string ) => ( {
 	siteTitle,
 } );
 
-export const setSiteVertical = ( siteVertical: SiteVertical ) => ( {
-	type: 'SET_SITE_VERTICAL' as const,
-	siteVertical,
-} );
-
-export const skipSiteVertical = () => ( {
-	type: 'SKIP_SITE_VERTICAL' as const,
-} );
-
 export const togglePageLayout = ( pageLayout: Template ) => ( {
```

Comment:
I searched and couldn't find any use of this action. It's sort of outside the scope of this PR, but if we remove this action then we can remove the `Template` type, which would remove the last remaining import from the vertical data store.
I don't know what `togglePageLayout` is for, but given it's something to do with a vertical template, I suspect it's not something we need.

Reply:
Good catch! Cleaned up in https://github.com/Automattic/wp-calypso/pull/59307/commits/889bba6464d574ff755d44d217aed5a3775a974e.